app/_how-tos/create-transit-gateway-with-operator-and-aws.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Create a Transit Gateway with {{ site.operator_product_name }} and AWS
+description: "Create a KonnectCloudGatewayTransitGateway resource with {{ site.operator_product_name }} and AWS."
+content_type: how_to
+
+
+breadcrumbs:
+  - /operator/
+  - index: operator
+    group: Konnect
+  - index: operator
+    group: Konnect
+    section: "Konnect CRDs: Cloud Gateways"
+
+products:
+  - operator
+
+works_on:
+  - konnect
+
+tags:
+  - aws
+
+prereqs:
+  inline: 
+    - title: AWS account
+      position: before
+      content: |
+          This tutorial requires an AWS account with permissions to create transit gateways and resource shares.
+      icon_url: /assets/icons/aws.svg
+    - title: AWS CLI
+      position: before
+      include_content: prereqs/aws-cli
+      icon_url: /assets/icons/aws.svg
+
+  operator:
+    konnect:
+      auth: true
+      control_plane: true
+      network: true
+
+tldr:
+  q: How can I create an AWS transit gateway and link it to {{site.konnect_short_name}} using {{ site.operator_product_name }}?
+  a: Create a transit gateway in AWS and create a resources share to share the transit gateway with the AWS account linked to your {{site.konnect_short_name}} account, then create a [`KonnectCloudGatewayTransitGateway`](/operator/reference/custom-resources/#konnectcloudgatewaytransitgateway) and accept the transit gateway attachment in AWS.
+
+faqs:
+  - q: Can I create a {{site.konnect_short_name}} Transit Gateway linked to an Azure virtual network?
+    a: Yes, refer to [Azure peering](/dedicated-cloud-gateways/azure-peering/) to learn how to configure your VNET Peering App on Azure, then configure the [`KonnectCloudGatewayTransitGateway`](/operator/reference/custom-resources/#konnectcloudgatewaytransitgateway) resource with the [`azureTransitGateway`](/operator/reference/custom-resources/#azuretransitgateway) field.
+
+related_resources:
+  - text: AWS Transit Gateway peering
+    url: /dedicated-cloud-gateways/transit-gateways/
+  - text: Azure peering
+    url: /dedicated-cloud-gateways/azure-peering/
+  - text: Dedicated Cloud Gateways
+    url: /dedicated-cloud-gateways/
+---
+
+## Create a transit gateway in AWS
+
+Use this command to create a transit gateway in AWS:
+```sh
+aws ec2 create-transit-gateway
```

Comment:
I got this error when I tested this:

`An error occurred (AuthFailure) when calling the CreateTransitGateway operation: AWS was not able to validate the provided access credentials`

I verified my credentials matched those in our AWS - Kong account. The region `us-east-2` matches what AWS has and I made sure to change the KonnectCloudGatewayNetwork resource to match. I think it might be a permissions issue because I think I had the same problem when I was testing this for Angel before. I think my account must not have access to Transit Gateways in AWS.